src/lib.rs

```diff
@@ -50,21 +51,22 @@
 //!
 //! ## Support for WebAssembly and asm.js
 //!
-//! The three Emscripten targets `asmjs-unknown-emscripten`,
-//! `wasm32-unknown-emscripten` and `wasm32-experimental-emscripten` use
-//! Emscripten's emulation of `/dev/random` on web browsers and Node.js.
-//!
-//! The bare WASM target `wasm32-unknown-unknown` tries to call the javascript
-//! methods directly, using either `stdweb` or `wasm-bindgen` depending on what
-//! features are activated for this crate. Note that if both features are
-//! enabled `wasm-bindgen` will be used. If neither feature is enabled,
-//! compiling `getrandom` will result in a compilation error. It can be avoided
-//! by enabling the `dummy` feature, which will make `getrandom` to use an
-//! always failing implementation.
+//! Getrandom supports all of Rust's current `wasm32` targets, and it works with
+//! both Node.js and web browsers. The three Emscripten targets
+//! `asmjs-unknown-emscripten`, `wasm32-unknown-emscripten`, and
+//! `wasm32-experimental-emscripten` use Emscripten's `/dev/random` emulation.
+//! The WASI target `wasm32-wasi` uses the [`__wasi_random_get`][17] function
+//! defined by the WASI standard.
 //!
-//! The WASI target `wasm32-wasi` uses the `__wasi_random_get` function defined
-//! by the WASI standard.
+//! Getrandom also supports `wasm32-unknown-unknown` by directly calling
+//! javascript methods. Rust currently has two ways to do this: [bindgen] and
+//! [stdweb]. Getrandom can using either by enabling the `wasm-bindgen` or
```

Comment:
can using

Reply:
Fixed, also make sure to correctly capitalize JavaScript